### DISCUSSION TITLE
ZuluSCSI Mini/Powerbook termination?

### DISCUSSION THREAD

**Discussion by SweJack** [ZuluSCSI/ZuluSCSI-firmware#47] [Q&A] [answered]
Hello, 

Been looking around for the information but haven't found a definitive answer. 

Is there any possibility of enabling/disabling termination on the ZuluSCSI Mini and/or Powerbook boards? 

Why? - I am having odd issues connecting a Mini Externally to my Powerbook in combination with a Powerbook edition internally. 
The Mini version works perfectly with an original SCSI drive internally.

**Comment by aperezbios** [comment, marked as answer]
No, there's no way to disable termination on the ZuluSCSI Mini (and the PowerBook edition, which is Mini-based), as those resistors are soldered in-place. Sorry.
